frontend/src/pages/notebookController/screens/admin/useRouteForNotebook.ts

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { getRoute } from '#~/api';
+
+// separate from useRouteForNotebook within projects directory. This one does not poll
+const useRouteForNotebook = (
```

Comment:
it may be better to reuse the current useRouteForNotebook with polling but add an option to make polling optional. While you aren there you can simplify the polling to use our useFetch hook which has a built in option to enable polling